app/views/users/getting_started.haml

```diff
@@ -4,8 +4,13 @@
       .hero-unit
         %h1.text-center
           = t('.well_hello_there')
+
         %h3.text-center
           = t(".community_welcome")
+          %h3.text-center
+            = link_to  "Get started with diaspora* tutorials!", "https://diasporafoundation.org/tutorials"
```

Comment:
- Layout/ExtraSpacing: Unnecessary spacing detected.
- Layout/SpaceBeforeFirstArg: Put one space between the method name and the first argument.